### PR TITLE
Fix issue 109

### PR DESCRIPTION
**Pull request by acdenisSK**

### CODE REVIEW

File: src/model/guild/member.rs [hunk 1]
```diff
@@ -352,6 +352,27 @@ impl Member {
...
+    /// println!("The permission bits for the member are: {}", member.permissions().expect("permissions").bits);
+    /// ```
+
+    /// # Errors
+    ///
+    /// Returns a [`ModelError::GuildNotFound`] if the guild the member's in could not be
+    /// found in the cache.
+    ///
+    /// [`ModelError::GuildNotFound`]: enum.ModelError.html#variant.GuildNotFound
+    pub fn permissions(&self) -> Result<Permissions> {
```
Comment (zeyla) [inline src/model/guild/member.rs]: This needs to be gated behind the `cache` feature.

File: src/model/guild/member.rs [hunk 2]
```diff
@@ -352,6 +352,27 @@ impl Member {
...
+        let dummy = self.find_guild()?.find().unwrap();
```
Comment (zeyla) [inline src/model/guild/member.rs]: Needs updating for b4bd771, needs to use the structfield `guild_id` instead of the method `find_guild` now.

**Comment by acdenisSK** [comment]
Regarding the two "merge branch ...", i messed up a bit in the first one since i forgot to update my local master.

**Comment by zeyla** [comment]
Thanks

**Pull request merged**